src/back/routes/debug.ts

```diff
@@ -21,9 +22,12 @@ export default routes((router) => {
   )
   router.post('/debug/report-error', withAuth, handleAPI(reportClientError))
   router.post('/debug/trigger', withAuth, handleAPI(triggerFunction))
+  router.delete('/debug/invalidate-cache', withAuth, handleAPI(invalidateCache))
 })
 
 function reportClientError(req: WithAuth): void {
+  const user = req.auth!
+  validateDebugAddress(user)
```

Comment:
I think this is used in other places than the debug section 🤔 And it shouldn't be under auth.